packages/components/src/tools-panel/styles.ts

```diff
@@ -19,6 +19,27 @@ export const ToolsPanel = css`
 	row-gap: ${ space( 6 ) };
 `;
 
+/**
+ * Items injected into a ToolsPanel via a virtual bubbling slot will require
+ * an inner dom element to be injected. The following rule allows for the
+ * CSS grid display to be re-established.
+ */
+export const ToolsPanelWithInnerWrapper = css`
```

Comment:
More of a personal preference, but I thought it'd be neat to extract/refactor the main common grid styles:

```diff
diff --git a/packages/components/src/tools-panel/styles.ts b/packages/components/src/tools-panel/styles.ts
index ecd0b6346c..220db2c5ef 100644
--- a/packages/components/src/tools-panel/styles.ts
+++ b/packages/components/src/tools-panel/styles.ts
@@ -9,14 +9,29 @@ import { css } from '@emotion/react';
 import { COLORS, CONFIG } from '../utils';
 import { space } from '../ui/utils/space';
 
+const toolsPanelGrid = {
+	container: css`
+		column-gap: ${ space( 4 ) };
+		display: grid;
+		grid-template-columns: 1fr 1fr;
+		row-gap: ${ space( 6 ) };
+	`,
+	item: {
+		halfWidth: css`
+			grid-column: span 1;
+		`,
+		fullWidth: css`
+			grid-column: span 2;
+		`,
+	},
+};
+
 export const ToolsPanel = css`
+	${ toolsPanelGrid.container };
+
 	border-top: ${ CONFIG.borderWidth } solid ${ COLORS.gray[ 200 ] };
-	column-gap: ${ space( 4 ) };
-	display: grid;
-	grid-template-columns: 1fr 1fr;
 	margin-top: -1px;
 	padding: ${ space( 4 ) };
-	row-gap: ${ space( 6 ) };
 `;
 
 /**
@@ -26,11 +41,8 @@ export const ToolsPanel = css`
  */
 export const ToolsPanelWithInnerWrapper = css`
 	> div {
-		column-gap: ${ space( 4 ) };
-		display: grid;
-		grid-template-columns: 1fr 1fr;
-		row-gap: ${ space( 6 ) };
-		grid-column: span 2;
+		${ toolsPanelGrid.container }
+		${ toolsPanelGrid.item.fullWidth }
 	}
 `;
 
@@ -45,7 +57,7 @@ export const ToolsPanelHeader = css`
 	display: flex;
 	font-size: inherit;
 	font-weight: 500;
-	grid-column: span 2;
+	${ toolsPanelGrid.item.fullWidth }
 	justify-content: space-between;
 	line-height: normal;
 
@@ -68,10 +80,10 @@ export const ToolsPanelHeader = css`
 `;
 
 export const ToolsPanelItem = css`
-	grid-column: span 2;
+	${ toolsPanelGrid.item.fullWidth }
 
 	&.single-column {
-		grid-column: span 1;
+		${ toolsPanelGrid.item.halfWidth }
 	}
 
 	/* Clear spacing in and around controls added as panel items. */
```

Reply:
Looks good. I'll make that happen, thanks.